server/clip_server/model/model.py

```diff
@@ -277,11 +278,17 @@ def __init__(
         scale_heads: bool = False,
         scale_attn: bool = False,
         scale_fc: bool = False,
+        use_flash: bool = True,
```

Comment:
I would like to take `flash_attention` as the default behavior by detecting the depends.

Reply:
So, can we have a helper function to detect whether flash_attention is usable?